pkg/k8s/watchers/pod.go

```diff
@@ -191,6 +191,13 @@ func (k *K8sWatcher) podsInit(k8sClient kubernetes.Interface, asyncControllers *
 	}
 }
 
+func skipHostNetworkedPodEvents(pod *slim_corev1.Pod) bool {
```

Comment:
This does not seem addressed from the previous review.

Reply:
@aanm The review feedback has already been addressed as the function is renamed. Also, note that the function returns a bool to indicate if the pod events need to be skipped or not. Since you didn't give any suggestions as to what the function should be renamed to, I used my best judgement.

Reply:
done